### ISSUE TITLE
Wrong parameter description.

### ISSUE THREAD

**Issue by krispenner** [AzureAD/microsoft-identity-web#1677]
https://github.com/AzureAD/microsoft-identity-web/blob/93e06894eff32179f5bbc1f37375eac6a12f7e2c/src/Microsoft.Identity.Web/WebAppExtensions/MicrosoftIdentityWebAppServiceCollectionExtensions.cs#L30

Change this:
```cs
/// (by default OpenIdConnectDefaults.AuthenticationScheme).</param> 
```

To this:
```cs
/// (by default CookieAuthenticationDefaults.AuthenticationScheme).</param> 
```

**Comment by jennyf19** [comment]
Thanks @krispenner

**Comment by jennyf19** [comment]
Thanks @krispenner

**Comment by jennyf19** [comment]
Released in [1.24.0](https://github.com/AzureAD/microsoft-identity-web/releases/tag/1.24.0).
